### PR TITLE
requesting a new test instance with correct protocol

### PR DESCRIPTION
**Pull request by jaybee84**
The last instance PR did not work so putting in a new PR for a new instance. This is all for testing purpose.

### CODE REVIEW

File: config/prod/newinstance_1.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2.yaml
+stack_name: iwanna-instance
```
Comment (zaro0508) [inline config/prod/newinstance_1.yaml]: better name please.  also we prefer it to match the filename.

File: config/prod/newinstance_1.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
```
Comment (zaro0508) [inline config/prod/newinstance_1.yaml]: is this your department and project?

File: config/prod/newinstance_1.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "joe.smith@sagebase.org"
```
Comment (zaro0508) [inline config/prod/newinstance_1.yaml]: not your email

File: config/prod/newinstance_1.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2.yaml
+stack_name: jb-new-instance-1
```
Comment (zaro0508) [inline config/prod/newinstance_1.yaml]: could you please change filename to match stack_name?  thanks.

Comment (jaybee84) [reply config/prod/newinstance_1.yaml]: Done!

**Pull request merged**